src_pp/sec_double_int_polar.ptx

```diff
@@ -1,487 +1,465 @@
+      <idx>
+        <h>double integral</h><h>in polar</h>
+      </idx>
 
-            \addplot [{\colortwo}, thick,smooth,domain=30:45,samples=30] ({cos(x)*(.8)},{sin(x)*(.8))});
-
-            \addplot [{\colortwo}, thick,smooth,domain=30:45,samples=30] ({cos(x)*(.8)},{sin(x)*(.8))});
-
-            \addplot [{\colortwo}, thick,smooth,domain=30:45,samples=30] ({cos(x)*(.6)},{sin(x)*(.6))});
-
-            \addplot [{\colortwo},thick, smooth,domain=0:.8,samples=2] ({cos(30)*(x)},{sin(30)*(x)});
-
-            \addplot [{\colortwo},thick, smooth,domain=0:.8,samples=2] ({cos(45)*(x)},{sin(45)*(x)});
-
-            \draw [rotate=30] (axis cs:.31,-.12) node [rotate=30] {$\underbrace{\rule{92pt}{0pt}}_{r_1}$};
-
-            \draw [rotate=45] (axis cs:.41,-.03) node [rotate=45] {$\overbrace{\rule{120pt}{0pt}}^{r_2}$};
-
-            \draw [thick,->,>=stealth,rotate=24] (axis cs:.45,0) arc (0:13:80pt);
-            \draw (axis cs: .4,.3) node {\scriptsize $\Delta \theta$};
-            \end{axis}
-
-            %\node [right] at (myplot.right of origin) {\scriptsize $0$};
-            %\node [above] at (myplot.above origin) {\scriptsize $\pi/2$};
-            \end{tikzpicture}
-            ]]>
-            </latex-image>
-          </image>
-        <!-- figures/fig_doublepol_intro_b.tex END -->
-        </figure>
-      </sidebyside>
-
-    </figure>
-
-    <p>
-      Now consider representing a region <m>R</m> with polar coordinates.
-      Consider <xref ref="fig_double_pol_intro">Figure</xref>(a).
-      Let <m>R</m> be the region in the first quadrant bounded by the curve.
-      We can approximate this region using the natural shape of polar coordinates:
-      portions of sectors of circles.
-      In the figure, one such region is shaded,
-      shown again in part (b) of the figure.
-    </p>
-
-    <p>
-      As the area of a sector of a circle with radius <m>r</m>,
-      subtended by an angle <m>\theta</m>,
-      is <m>A = \frac12r^2\theta</m>,
-      we can find the area of the shaded region.
-      The whole sector has area <m>\frac12r_2^2\Delta \theta</m>,
-      whereas the smaller,
-      unshaded sector has area <m>\frac12r_1^2\Delta \theta</m>.
-      The area of the shaded region is the difference of these areas:
       <me>
-        \Delta A_i = \frac12r_2^2\Delta\theta-\frac12r_1^2\Delta\theta = \frac12\big(r_2^2-r_1^2\big)\big(\Delta\theta\big) = \frac{r_2+r_1}{2}\big(r_2-r_1\big)\Delta\theta
+        \iint_Rf(x,y)\ dA = \int_\alpha^\beta\int_{g_1(\theta)}^{g_2(\theta)} f\big(r\cos(\theta) ,r\sin(\theta) \big)\ r\ dr\ d\theta
```

Comment:
This doesn't look right. I think git is getting confused by the size of the diff here.
Line numbers are off, and one equation is getting replaced by another.

Reply:
The file looks fine though: everything is in place there.
So maybe it's just not displaying the diff correctly?